lib/internal/modules/helpers.js

```diff
@@ -392,6 +392,24 @@ ObjectFreeze(compileCacheStatus);
 const constants = { __proto__: null, compileCacheStatus };
 ObjectFreeze(constants);
 
+async function resolveLoadAndCache(specifier, base = 'data:', importAttributes = kEmptyObject, conditions) {
+  const cascadedLoader = require('internal/modules/esm/loader').getOrInitializeCascadedLoader();
+  // TODO: this should hit the cache (and populate it if it finds nothing)
```

Comment:
This is already doing that, no? I see gets and sets

Reply:
Or, it appears to do for `ModuleLoader` but not `CustomizedModuleLoader`. Is that what you mean?